docs/serving/revisions/revision-admin-config-options.md

```diff
@@ -1,3 +1,73 @@
 # Administrator configuration options
 
 If you have cluster administrator permissions for your Knative installation, you can modify ConfigMaps to change the global default configuration options for Revisions of Knative Services on the cluster.
+
+## Garbage collection
+
+--8<-- "about-revisions-garbage-collection.md"
+
+You can set cluster-wide garbage collection configurations for your cluster by modifying the `config-gc` ConfigMap.
+
+The following garbage collection settings can be modified:
+
+Name|Description
+-|-
+`retain-since-create-time`| The time that must have elapsed since a Revision was created before a Revision is considered for garbage collection.
+`retain-since-last-active-time`| The time that must have elapsed since a Revision was last active before a Revision is considered for garbage collection.
+`min-non-active-revisions`| The minimum number of inactive Revisions to retain.
+`max-non-active-revisions`| The maximum number of inactive Revisions to retain.
+
+Revisions are always retained if they belong to any one of the following categories:
+
+- The Revision is active and is being reference by a Route.
+- The Revision was created within the time specified by the `retain-since-create-time` setting.
+- The Revision was last referenced by a Route within the time specified by the `retain-since-last-active-time` setting.
+- There are fewer existing Revisions than the number specified by the `min-non-active-revisions` setting.
```

Comment:
fewer than min wording is confusing, maybe an example?

Reply:
Since this pretty much came from existing text that I have just moved / grouped / edited in the doc, I think this is maybe a little out of scope to add for this PR.
Maybe once this is merged a new issue could be opened to improve the examples, or add examples for all of these bullet points so that it's consistent?
Otherwise if it's really a deal-breaker and you can provide a suggestion for an example here I can add it?

Reply:
not a deal breaker, I can work on a later PR with suggestions for improvements